doc/main.html

```diff
@@ -1072,7 +1072,16 @@ <h4>Formulae</h4>
 
       </section>
 
+      <section id="sec-special-characters">
+        <h4>Special Characters</h4>
+        <p>Characters should should be encoded as UTF-8-encoded Unicode codepoints, e.g. あ, instead of HTML entities, e.g. <code>&amp;#x3042;</code>, expect as needed for usage in <code>pre</code> or examples.
```

Comment:
```suggestion
        <p>Characters should should be encoded as UTF-8-encoded Unicode codepoints, e.g. あ, instead of HTML entities, e.g. <code>&amp;#x3042;</code>, except as needed for usage in <code>pre</code> or examples.
```